resources/assets/helpers.js

```diff
@@ -281,3 +289,19 @@ export function env(key) {
  */
 export const camelCaseKeys = object =>
   mapKeys(object, (value, key) => camelCase(key));
+
+/**
+ * Merge paginated GraphQL queries.
+ *
+ * @param {*} previous
+ * @param {*} param1
+ */
+export const updateQuery = (previous, { fetchMoreResult }) => {
+  return mergeWith({}, previous, fetchMoreResult, (dest, src) => {
+    // By default, Lodash's `merge` would try to merge *each* array
+    // item (e.g. `edges[0]` with then next page's `edges[0]`).
+    if (isArray(dest)) {
+      return [...dest, ...src];
+    }
+  });
+};
```

Comment:
@DFurnes Would you mind articulating how this works? 😖 sorry. I'm having trouble grokking this offhand

Reply:
Yeah it's kinda wonky! By default, Lodash's `merge` has a kinda weird way of merging arrays (it'll merge each element in the array with the corresponding one in the new array). So, by default, we'd do something like this:

```js
merge([item1, item2, item3], [item4, item5, item6]);
// => [merge(item1, item4), merge(item2, item5), merge(item3, item6)];
```

That's obviously not what we want here though! 😂 This "customizer" function just overrides Lodash's "merge strategy" for arrays, telling it to instead concatenate them:

```js
merge([item1, item2, item3], [item4, item5, item6]);
// => [item1, item2, item3, item4, item5, item6];
```

Reply:
Aha lol, fair, thanks for clarifying! (And I should have taken a peak at [the documentation](https://lodash.com/docs/4.17.15#mergeWith) for `mergeWith` where this is literally their provided example 😅(though they're using `Arrya.concat`, any specific reason you diverged, or just cuz?))

Reply:
I just thought that spreading the arrays read a little clearer (since with `Array.concat` it's kinda unclear which is being concatenated to which at first glance).